opal/mca/btl/usnic/btl_usnic_recv.c

```diff
@@ -142,7 +143,10 @@ void opal_btl_usnic_recv_call(opal_btl_usnic_module_t *module,
                         (unsigned)bseg->us_btl_header->payload_len,
                         (int)bseg->us_btl_header->tag);
 #endif
-            reg->cbfunc(&module->super, hdr->tag, &seg->rs_desc, reg->cbdata);
+            seg->rs_desc.endpoint = endpoint;
```

Comment:
Why are these in the seg instead of being on the stack as for the other BTLs ?

Reply:
Not sure. Seems to be a design choice for the usnic btl. They were on the stack before this change so I will resolve this. @jsquyres can probably give more information.